### PR TITLE
chore: add dusk-3 details

### PR DESCRIPTION
**Pull request by joroshiba**

### CODE REVIEW

File: dusk-3/evm/genesis.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "homesteadBlock": 0,
+      "eip150Block": 0,
+      "eip155Block": 0,
+      "eip158Block": 0,
+      "byzantiumBlock": 0,
+      "constantinopleBlock": 0,
+      "petersburgBlock": 0,
+      "istanbulBlock": 0,
+      "berlinBlock": 0,
+      "londonBlock": 0,
```
Comment (jbowen93) [inline dusk-3/evm/genesis.json:13]: Did we want to upgrade the EVM to be on the shanghai fork or w/e?

File: dusk-3/sequencer/genesis.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        "type": "tendermint/PubKeyEd25519",
+        "value": "ixbsw2r9ME9ikcWTwWSdtk+AxNNQ4PyzOjNnyiMMe58="
+      },
+      "power": "1",
+      "name": "node3"
+    }
+  ],
+  "app_hash": "",
+  "app_state": {
+    "accounts": [
```
Comment (jbowen93) [inline dusk-3/sequencer/genesis.json]: These balances aren't the validators right, just the faucets and such? Is there any reason the validators would need to have a balance at genesis?

**Review by jbowen93 — APPROVED** [review]

File: dusk-3/sequencer/genesis.json (hunk 2 above)
Comment (joroshiba) [reply dusk-3/sequencer/genesis.json]: Correct, the validators have no account balance. These are accounts used for faucets.

File: dusk-3/evm/genesis.json (hunk 1 above)
Comment (joroshiba) [reply dusk-3/evm/genesis.json:13]: I don't have that enabled right now.... doh.

Will sync a node locally and see if I can reach same state with it set.

Comment (joroshiba) [reply dusk-3/evm/genesis.json:13]: Don't have this enabled right now, can't enable from start. We can enable the fork in the future.

I missed a file update in the rush here.

**Pull request merged**